bubble_sort.rb

```diff
@@ -0,0 +1,25 @@
+def bubble_sort(arr)
+  yield(arr) if block_given?
+  arr
+end
```

Comment:
If you are yielding, then your method is not doing anything. This is more likely to be reserved for the bubble_sort_by with some modifications.